ui/src/app/pages/workspace/workspace-edit-text.tsx

```diff
@@ -38,10 +38,18 @@ export const toolTipTextStigmatization = <div>
 
 
 export const ResearchPurposeDescription =
-    <div style={{display: 'inline'}}>The <i>All of Us</i> Research Program requires each user
-      of <i>All of Us</i> data to provide a meaningful description of the intended purpose of data
-      use for each workspace they create. Your responses will not be used to make decisions about
-      data access.</div>;
+    <div style={{display: 'inline'}}>The <AouTitle/> requires each data user of the
```

Comment:
There's a missing space here. (look at the screenshot. I think the space after `the` is missing.)